src/Subscriptions/Storage/CacheStorageManager.php

```diff
@@ -34,16 +36,26 @@ class CacheStorageManager implements StoresSubscriptions
     /**
      * The time to live for items in the cache.
      *
-     * @var \DateInterval|int|null
+     * @var int|null
      */
     protected $ttl;
 
     public function __construct(CacheManager $cacheManager)
     {
-        $this->cache = $cacheManager->store(
-            config('lighthouse.subscriptions.storage', 'file')
-        );
-        $this->ttl = config('lighthouse.subscriptions.storage_ttl');
+        /** @var \Illuminate\Contracts\Config\Repository $config */
+        $config = app(ConfigRepository::class);
+
+        $storage = $config->get('lighthouse.subscriptions.storage') ?? 'file';
+        if (! is_string($storage)) {
+            throw new Exception('Config setting lighthouse.subscriptions.storage must be a string or `null`, got: '.\Safe\json_encode($storage));
```

Comment:
Again, I think we should consider creating a custom exception here or at least make it a runtimeException 👍